README.md

```diff
@@ -124,6 +166,9 @@ SQL database. If you need an example, you can use: people and wishes
 (one-to-many).
 
 Your answer:
+A database schema is a way to logically group objects such as tables, views, stored procedures etc.
+And means one parent can have many children.
+
 ```
```

Comment:
The Schema defines the tables and columns of our database, i.e. the structure of our database